integration-tests/tests/list_test.go

```diff
@@ -59,7 +59,7 @@ func (s *listSuite) TestListMustPrintAppVersion(c *check.C) {
 	expected := "(?ms)" +
 		"Name +Version +Rev +Developer *\n" +
 		".*" +
-		"^hello-world +(\\d+)(\\.\\d+)* +[0-9]+ +.* *\n" +
+		"^hello-world +(\\d+)(\\.\\d+)* +[x0-9]+ +.* *\n" +
```

Comment:
In this particular case the snap comes from the store so we know that there will be no `x` in here. However its not the main purpose of the test and certainly not a blocker if it gets added.